Examples/AferoLab/AferoLab/Models/DeviceCollectionCollator.swift

```diff
@@ -185,11 +185,9 @@ class DeviceCollectionDeviceCollator: DeviceCollectionObserving, DeviceCollator
     func updateCollatedIds() {
         collatedDeviceIds = deviceCollection?
             .devices
-            .filter {
-                $0.isPresentable || $0.isLocalSofthub
```

Comment:
This does change the meaning a little bit, in t hat it will cause us to show any device w/o presentation. Probably intent, but just wanted to make sure.

Reply:
deviceCollecton.devices already filters out the devices that do not have presentation, thanks for checking